More Asphalt 5 docs updates (#140)
asphalt-framework/asphalt: README.rst

  :target: https://asphalt.readthedocs.io/en/latest/?badge=latest
  :alt: Documentation Status

> Asphalt is an asynchronous microframework for network oriented applications.

Reviewer: Do we still need to mention "network oriented applications"? AFAIK Asphalt is "just" an asynchronous microframework.

Author: How would you prefer to word this?

Reviewer: Maybe:
```suggestion
Asphalt is a microframework for building applications and orchestrating asynchronous services.
```

Reviewer: > But how about "Asphalt is an asynchronous application microframework"?

That works too, I'm just thinking it's too bad we don't emphasize what it's good at: starting services and tearing them down, in a "safe" way.
What do you you think about replacing "orchestrating" by "managing"?

Author: Trying to shoehorn its complex capabilities into a single sentence is probably counterproductive. If we're to advertise those, it should be done in more detail later in the document.

Reviewer: OK, then "Asphalt is an asynchronous application microframework" is a good one-sentence description :+1:

Author: Updated.
